lib/core/models/message_model.dart

```diff
@@ -25,40 +29,55 @@ class MessageModel {
   @HiveField(6)
   String? id;
   @HiveField(7)
-  final Map<String, MessageState> userStates; // userID -> state of the message
+  final String? photo;
   @HiveField(8)
-  final bool isPinned;
+  Uint8List? photoBytes;
```

Comment:
we don't need photos nor photoBytes in the messageModel.
if there is a media, it is handled in the MessageContent